src/cdk/v2/destinations/the_trade_desk/transformRecord.js

```diff
@@ -40,6 +40,9 @@ const batchResponseBuilder = (items, config) => {
   return response;
 };
 
+const checkNullUndefinedEmptyFields = (fields) =>
+  !!Object.entries(fields).some(([, value]) => !value);
```

Comment:
`!value` can create a problem if the field is a boolean or number with `0` as value.
It would be better if we use `isDefinedAndNotNull` kind of util function to achieve this.

Reply:
Trade desk doesn't support falsy values (0, boolen, number). They support `GUID`. So `!value` is valid check here.